docs/source/standard.md

```diff
@@ -255,39 +255,41 @@ The ability of the organisation to support users who want to browse the data ava
-### 4.1 Vulnerability management
+### 4.1 Vulnerability management and security testing
 
-The ability of the organisation to identify, assess, report on, manage and remediate cyber vulnerabilities across endpoints, workloads, and systems.
+Vulnerability management describes the ability of the organisation operating a TRE to identify, assess, report on, manage and remediate cyber vulnerabilities across endpoints, workloads, and systems. Security testing enables the organisation to gain assurance in the security of a TRE by testing or attempting to breach some or all of that system's security.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+| Statement                                                                                                                                   | Guidance                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Regular vulnerability scans of TRE infrastructure should be conducted                                                                       | Ensuring that scans are done on a regular basis can enable TRE operators can identify and address weaknesses that may have been introduced during the TRE's operational lifetime. One way this can be achived is by ensuring that all machines (whether physical or virtual/cloud-based) are set to check for operating system updates on a regular basis, and install any security patches pushed by the developers of those operating systems. |
+| TREs should undergo compliance checking of machine and resource configurations to ensure adherence to security standards and best practices | Adhering to the highest level of security standards suggested by the IT department of the TRE operators organisation will reduce the risk of security breaches and data leakage. This should be done in advance of any sensitive data being uploaded to the TRE.                                                                                                                                                                                 |
```

Comment:
Are there any particular security standard or best practices we should mention here? Delegating this to "the IT department" makes it sound like we don't have any thoughts on this.

Reply:
Reworded